src/controllers/PendingTransactionController.ts

```diff
@@ -0,0 +1,52 @@
+
+import { Request, Response } from "express";
+import { sendJSONresponse } from "../common/Utils";
+import { Config } from "../common/Config";
+import { Error } from "mongoose";
+let Promise = require("bluebird");
+
+export class PendingTransactionController {
+
+    public readPendingTransaction(req: Request, res: Response) {
```

Comment:
this should be called `getPendingTransactions`

Reply:
Renamed